conf/modules.config

```
@@ -626,6 +626,36 @@ process {
         ]
     }
 
+    withName: ARGNORM_ABRICATE {
+        publishDir = [
+            path: {"${params.outdir}/arg/hamronization/abricate/argnorm/"},
```

```suggestion
            path: {"${params.outdir}/arg/argnorm/abricate/"},
```

should be set for all ARGNORM modules